src/sshRunner.ts

```diff
@@ -56,6 +56,8 @@ export class SSHRunner extends TerminalBaseRunner {
     }
 
     protected async runAnsibleInTerminal(playbook, cmds, terminalId: string): Promise<void> {
+        TelemetryClient.sendEvent('ssh');
```

Comment:
Can we get the class name in runtime and send the event in BaseRunner. Once there are new running method, contributor doesn't need to care about the telemetry.

Reply:
agree, that's what i'm thinking also besides the quick fix.

Reply:
after second think and re-visit previous telemetry logic, i'd like to keep telemetry sending in each child runner instead of baserunner. Because child runner could customize event sent, and when to send. eg. in cloudshell, we'll send telemetry at triggering, error, succeed.